### DISCUSSION TITLE
Logpoint PySigma Backend

### DISCUSSION THREAD

**Discussion by swachchhanda000** [SigmaHQ/pySigma#251] [General]
Hi,  We at Logpoint have been working on the logpoint-pysigma-backend for quite some time now. I would like to seek your help and collaboration for this release.
I hope you understand my concerns.
Thanks

**Comment by thomaspatzke** [comment]
What are the concrete issues or questions?

**Comment by swachchhanda000** [comment]
We are ready with the backend for publishing. and we want this pipeline to be integrated into Sigma CLI.

**Comment by sifex** [comment]
Hey, once the repo is published, you can submit a PR to the https://github.com/SigmaHQ/pySigma-plugin-directory/ repository with the update to https://github.com/SigmaHQ/pySigma-plugin-directory/blob/main/pySigma-plugins-v1.json 

From there, once merged, everyone should be able to download the backend. Hope this helps ☺️

**Comment by swachchhanda000** [comment]
I have just send the PR @sifex. Please have a look.
https://github.com/logpoint/pySigma-backend-logpoint
cc @thomaspatzke
